alpha/apps/kaltura/modules/extwidget/actions/embedIframeJsAction.class.php

```diff
@@ -77,7 +77,9 @@ public function execute()
 		{
 			if (!$iframeEmbed)
 				$host = "$protocol://". kConf::get('html5lib_host') ."/";
-			$html5_version = kConf::get('html5_version');
+			$embedPlaykitConf = kConf::getMap(kConfMapNames::APP_VERSIONS);
```

Comment:
Better approach in one line - 
kConf::getArrayValue('html5_version','playerApps',kConfMapNames::APP_VERSIONS,null);

Signature - getArrayValue($paramName, $sectionName, $mapName = 'local', $defaultValue = false)